pandas/tests/tseries/test_frequencies.py

```diff
@@ -1,793 +0,0 @@
-from datetime import datetime, timedelta
-
-import numpy as np
-import pytest
-
-from pandas._libs.tslibs import frequencies as libfrequencies, resolution
-from pandas._libs.tslibs.ccalendar import MONTHS
-from pandas._libs.tslibs.frequencies import (
-    INVALID_FREQ_ERR_MSG, FreqGroup, _period_code_map, get_freq, get_freq_code)
-import pandas.compat as compat
-from pandas.compat import is_platform_windows, range
-
-from pandas import (
-    DatetimeIndex, Index, Series, Timedelta, Timestamp, date_range,
-    period_range)
-from pandas.core.tools.datetimes import to_datetime
-import pandas.util.testing as tm
-
-import pandas.tseries.frequencies as frequencies
-import pandas.tseries.offsets as offsets
-
-
-class TestToOffset(object):
-
-    def test_to_offset_multiple(self):
```

Comment:
Any chance you could provide some direction on how this has been split up? I see this test is now just `test_to_offset` and is parametrized which is great, but does that new test also include elements from other tests?

Reply:
Absolutely.  I renamed this one because it now is more generic to include elements from other tests that were essentially testing the `to_offset`.

Part of the methodology in these refactors to identify how we can:

* Make more modular tests
* Leverage more `pytest` idiom
* Re-organize test content so that similar tests are strewn all over the place.

Reply:
Gotcha. How hard would it be to parametrize in one PR and then move in a second PR? I think that would make review easier just not sure how feasible that is from your perspective

Reply:
The parameterization and reorganization often go hand-in-hand to an extent (e.g. you parameterize because you combine tests with similar execution but different inputs), so it's not straightforward to separate out.